code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -74,10 +76,48 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/mountain/Move()
+	var/turf/floor = get_turf(src)
+	for(var/thing in floor)
+		if(istype(thing, /obj/effect/decal/cleanable/blood))
+			var/obj/effect/decal/cleanable/blood/blood_decal = thing
+			adjustBruteLoss(-clamp((maxHealth*0.005)*(blood_decal.bloodiness*0.01), 0, maxHealth)) // Heal up to 0.5% heath from bloody ground.
+			QDEL_NULL(blood_decal)
+			belly += 0.01
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/organ))
+			var/obj/item/organ/floor_organ = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Organs
+			QDEL_NULL(floor_organ)
+			belly += 0.05
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/bodypart))
+			var/obj/item/bodypart/floor_part = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Body Parts
+			QDEL_NULL(floor_part)
+			belly += 0.05 // Progresses toward getting bigger
+			StageChange(TRUE)
+			continue
 	if(finishing)
 		return FALSE
+	new /obj/effect/decal/cleanable/old_flesh(floor)
 	return ..()
 
+/mob/living/simple_animal/hostile/abnormality/mountain/apply_damage(damage, damagetype, def_zone, blocked, forced, spread_damage, wound_bonus, bare_wound_bonus, sharpness, white_healable)
+	. = ..()
+	var/messyArea = list()
+	var/validArea = view(2, src)
+	for(var/I = 1 to pick(list(3, 4, 5)))
+		var/turf/space = pick_n_take(validArea)
+		if (!(space in messyArea) && !space.density)
+			messyArea += space
+		else
+			I -= 1
+	playsound(src, 'sound/effects/blobattack.ogg', 75)
+	for(var/turf/T in messyArea)
```

Comment:
This will honestly get annoying really fast with fast-firing things like beak.

Reply:
Hes supposed to be gross and annoying, and if you mean the sound then that was barely audible over normal attacks at 50 volume. But Mountain straight up making the place grosser is the ideal. His mess is cleanable easily with Sprayers and can actually hide blood for him, that he can then walk over and heal from